sierra_adapter/README.md

```diff
@@ -0,0 +1,288 @@
+# Sierra adapter
+
+This is the adapter for [Sierra][sierra], the library management system.
+
+This stack is responsible for reading records from Sierra and copying them into the pipeline.
+Since Sierra records can be edited at any time, it is also responsible for keeping the pipeline up-to-date with new edits or additions.
+
+See also:
+
+-   [Notes on Sierra IDs](https://github.com/wellcometrust/catalogue/blob/master/docs/sierra/sierra_ids.md)
+
+[sierra]: https://www.iii.com/products/sierra-ils/
+
+
+
+## Shape of the Sierra data
+
+The Sierra API contains multiple types of record.
+The two we're especially interested in are *bibs* and *items*.
+
+*   A *bib* is a *bibliographic record* – for example, an individual book.
+    It's the "concept" of the book, rather than an individual copy.
+*   An *item* is a retrievable object – for example, a physical copy of a book, or an electronic book that you can download.
+
+There is a many-to-many mapping between bibs and items:
+
+*   A single bib can be associated with multiple items: for example, we might have multiple physical copies (the items) of a book (the bib).
+*   A single item can be associated with multiple bibs: for example, i15699833 is a single physical item (a bound volume of patents), and each of the individual patents has its own bib number.
+
+<img style="width: 400px" src="images/shape_of_the_data.svg">
+
+When we create a Work from the Sierra data, we take a single bib and all the items it is associated with.
```

Comment:
I think it would be clearer to put this under a different heading (maybe even as an addendum), with some context that this creation of a work, and indeed the relationship illustrated in `work_from_sierra.svg`, are not things that are really performed or represented in the adapter.

Perhaps you could even remove this bit entirely / put it in a different document (eg a Sierra transformer document).

Reply:
I think it does belong here (at least part of it) – the Sierra adapter needs to produce the information that the transformer needs to create a Work. I've pulled it into a separate section.